src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -21,6 +21,7 @@
 #include "Tutorial.h"
 #include "Ui.h"
 #include "Ui/ToolManager.h"
+#include "Ui/Windows/Construction/Construction.h"
```

Comment:
I wonder, isn't this a bit excessive to include in the window manager unit? Maybe the construction tools could be handled from the tool manager instead.

Reply:
Yes it is a bit ridiculous but thought it would be fine for now